docs/reference/vpc-association-policy.md

```diff
@@ -0,0 +1,85 @@
+# VpcAssociationPolicy API Reference
+
+## VpcAssociationPolicy
+
+VpcAssociationPolicy is a CRD that can be attached to a Gateway to define the ServiceNetworkVpcAssociation configuration.
```

Comment:
Let's please clarify acronyms before using them in docs, e.g.

"VpcAssociationPolicy is a Custom Resource Definition (CRD) that can be attached to a Gateway to define the ServiceNetworkVpcAssociation configuration."